spec/integration/cache_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt_spec/config'
+require 'bolt_spec/integration'
+require 'bolt_spec/project'
+
+describe 'caching plugins' do
+  include BoltSpec::Config
+  include BoltSpec::Integration
+  include BoltSpec::Project
+
+  let(:cache_file) { File.join(project_path, '.plugin_cache.json') }
```

Comment:
You can just call `project.cache_file`, since `BoltSpec::Project` includes a method that returns the project.